apps/ledger-live-desktop/src/renderer/modals/Platform/Exchange/CompleteExchange/Body.tsx

```diff
@@ -187,27 +184,22 @@ const Body = ({ data, onClose }: { data: Data; onClose?: () => void | undefined
     ],
   );
 
-  useEffect(() => {
-    if (error) {
-      if (
-        ![
-          error instanceof WrongDeviceForAccount,
-          error instanceof CompleteExchangeError && error.message === "User refused",
-        ].some(Boolean)
-      ) {
-        onClose?.();
-      }
-    }
-  }, [onCancel, error, onClose]);
+  // useEffect(() => {
+  //   /**
+  //    * If we want to close the drawer automatically, we need to ensure onCancel is also called
+  //    * this will gives the "control" back to live app.
+  //    *
+  //    * On drawer manually closed, we send an error back ("Interrupted by user")
+  //    */
+  //   if ([error instanceof SOME_ERROR_WE_WANT_LIVE_APP_TO_HANDLE]) {
+  //     onCancel(error);
+  //     onClose(error)
+  //   }
+  // }, [onCancel, error]);
 
   useEffect(() => {
-    if (broadcastRef.current || !signedOperation) return;
-    console.log("[moonpay] about to broadcast");
-    broadcast(signedOperation)
-      .then(onBroadcastSuccess, setError)
-      .finally(() => {
-        broadcastRef.current = true;
-      });
+    if (!signedOperation) return;
```

Comment:
Can you confirm this with @lpaquet-ledger ?
```suggestion
   if (broadcastRef.current || !signedOperation) return;
```

Reply:
Yeah I added this because if not the useEffect could resend the same broadcast and create this error https://ledgerhq.atlassian.net/browse/LIVE-11708